backbone/src/main/java/org/researchstack/backbone/model/ConsentDocument.java

```diff
@@ -66,6 +68,29 @@
      */
     private String htmlReviewContent;
 
+    /**
+     * True if consent must require the user's name
+     */
+    @SerializedName("requiresName")
+    private boolean requiresName = true;
+    public boolean getRequiresName() { return requiresName; }
+    public void setRequiresName(boolean requiresName) { this.requiresName = requiresName; }
+
+    /**
+     * True if consent must require the user's birthdate
+     */
+    @SerializedName("requiresBirthdate")
+    private boolean requiresBirthdate = true;
+    public boolean getRequiresBirthdate() { return requiresBirthdate; }
+    public void setRequiresBirthdate(boolean requiresBirthdate) { this.requiresBirthdate = requiresName; }
+
+    /**
+     * True if consent must require the user's signature
+     */
+    @SerializedName("requiresSignature")
```

Comment:
Nit-picky comment: let's make the organization consistent. Either fields all at top, getter/setters below. Or grouped like you have it.